docs/Releases/v0.2.0-release.md

```diff
@@ -0,0 +1,364 @@
+---
+layout: default
+title: v0.2.0
+nav_order: 1
+parent: Releases
+---
+
+Hello again, world! The Operator-framework community is excited to announce the [v0.2.0](https://github.com/operator-framework/operator-controller/releases/tag/v0.2.0) release of operator-controller! 
+
+With this release of operator-controller, you can install an operator from a catalog of operators. This might sound familiar to exisiting users of OLM v0, however, there are some key differences that is evident in the user experience of installing an operator from a catalog of operators, even at this early stage of OLM v1. 
+
+Let us walk through an example of installing an operator from a catalog of operators, which will highlight the differences mentioned above. 
+
+First, let's get a [Kind](https://kind.sigs.k8s.io) cluster fired up 
+
+```bash
+$ kind create cluster 
+Creating cluster "kind" ...
+ ✓ Ensuring node image (kindest/node:v1.26.3) 🖼
+ ✓ Preparing nodes 📦  
+ ✓ Writing configuration 📜 
+ ✓ Starting control-plane 🕹️ 
+ ✓ Installing CNI 🔌 
+ ✓ Installing StorageClass 💾 
+Set kubectl context to "kind-kind"
+You can now use your cluster with:
+
+kubectl cluster-info --context kind-kind
+
+Have a question, bug, or feature request? Let us know! https://kind.sigs.k8s.io/#community 🙂
+```
+
+Next, let's install the v0.2.0 release of operator-controller
+
+```bash
+$ curl -L -s https://github.com/operator-framework/operator-controller/releases/download/v0.2.0/install.sh | bash -s
+.
+.
+deployment.apps/operator-controller-controller-manager created
+deployment.apps/operator-controller-controller-manager condition met
+```
+
+If we look into the cluster, we'll see that the installation process installed the various [components](components.md) of OLM v1: 
+
+```bash
+$ kubectl get namespaces 
+NAME                         STATUS   AGE
+catalogd-system              Active   2m4s
+cert-manager                 Active   3m28s
+crdvalidator-system          Active   3m1s
+default                      Active   17m
+kube-node-lease              Active   17m
+kube-public                  Active   17m
+kube-system                  Active   17m
+local-path-storage           Active   16m
+operator-controller-system   Active   82s
+rukpak-system                Active   3m1s
+```
+
+Looks like all the [components](/docs/components.md) `catalogd-system`, `operator-controller-system`, and `rukpak-system` are available in the cluster!
+
+Let's now add a repository (catalog) of operators to the cluster, to make the operators shipped in the catalog available for installation on the cluster. We'll use the upstream community-operators catalog [quay.io/operatorhubio/catalog](https://quay.io/repository/operatorhubio/catalog?tab=tags&tag=latest): 
```

Comment:
Nit:
```suggestion
Now let's add a catalog (similar to a repository in other package managers) of operators to the cluster, to make the operators shipped in the catalog available for installation on the cluster. We'll use the upstream community-operators catalog [quay.io/operatorhubio/catalog](https://quay.io/repository/operatorhubio/catalog?tab=tags&tag=latest): 
```

Reply:
Changed the sentence a bit more than the suggestion. PTAL.